frontend/public/locales/fr/apply-adult-child.json

```
@@ -281,6 +281,16 @@
         "has-social-insurance-number": "(FR) Select whether or not this child has a social insurance number",
         "is-parent": "(FR) Select whether or not you are this child's parent or legal guardian"
       }
+    },
+    "apply-for-yourself": {
+      "page-title": "(FR) Apply for yourself",
+      "ineligible-to-apply": "(FR) Your children are not currently eligible to apply for the Canadian Dental Care Plan. Only children under 18 are eligible at this time.",
+      "eligibility-info": "(FR) Find out when they can apply.",
+      "eligibility-info-href": "(FR) https://www.canada.ca/en/services/benefits/dental/dental-care-plan/apply.html#when",
```

```suggestion
      "eligibility-info-href": "https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires/demande.html#quand",
```

You can get these links by clicking on the english link and toggling language. 😄

---

Not needed since the route submit redirect to an internal route.